pkg/shell/base_index.js

```diff
@@ -575,20 +576,10 @@ function Index() {
 
     /* Watchdog for disconnect */
     const watchdog = cockpit.channel({ payload: "null" });
-    $(watchdog).on("close", function(event, options) {
+    watchdog.addEventListener("close", (event, options) => {
         const watchdog_problem = options.problem || "disconnected";
         console.warn("transport closed: " + watchdog_problem);
-        $(self).triggerHandler("disconnect", watchdog_problem);
-    });
-
-    /* Handles an href link as seen below */
-    $(document).on("click", "a[href]", function(ev) {
-        const a = this;
-        if (!a.host || window.location.host === a.host) {
-            self.jump(a.getAttribute('href'));
-            ev.preventDefault();
-            ev.stopImmediatePropagation();
-        }
```

Comment:
So this translated standard clicks into cockpit.jump() calls to track their history? This seems to have gone without replacement -- does that change anything wrt. history/back button handling?

Reply:
I believe that this used to be used for the main navigation. But that now is properly handled by components themselves, e.g. https://github.com/cockpit-project/cockpit/blob/main/pkg/shell/indexes.js#L341

I think it was forgotten in https://github.com/cockpit-project/cockpit/commit/377054a3f1189bb26122ed8c8b798cea8137729d